diff --git a/node_modules/onnxruntime-react-native/android/build.gradle b/node_modules/onnxruntime-react-native/android/build.gradle
index 41b4359..1b5e7e7 100644
--- a/node_modules/onnxruntime-react-native/android/build.gradle
+++ b/node_modules/onnxruntime-react-native/android/build.gradle
@@ -250,5 +250,0 @@ dependencies {
-  if (VersionNumber.parse(REACT_NATIVE_VERSION) < VersionNumber.parse("0.71")) {
-    extractLibs "com.facebook.fbjni:fbjni:+:headers"
-    extractLibs "com.facebook.fbjni:fbjni:+"
-  }
-
